scripts/test/launchTestUtilities.ts

```diff
@@ -17,8 +18,121 @@ const minimum = 'minimum'
 
 const disableWorkspaceTrust = '--disable-workspace-trust'
 
-export const integrationSuite = 'integration'
-export const e2eSuite = 'e2e'
+type SuiteName = 'integration' | 'e2e' | 'unit'
+
+/**
+ * This is the generalized method that is used by different test suites (unit, integration, ...) in CI to
+ * setup vscode and then run tests. An important thing to note is that in CI we do not have VS Code installed,
+ * so this script needs to do this itself.
+ *
+ * If you want to run the tests manually you should use the `Run & Debug` menu in VS Code instead.
```

Comment:
Why? I used `npm run test` heavily while debugging the "unresolved promise" issues. If that is no longer possible then that's a regression. CLI/terminal usage is important, it can really speed up debugging.

Reply:
`npm run test` still works, you just don't get debugging with it